api/src/wrangler.rs

```diff
@@ -123,8 +126,14 @@ impl ExecutionWrangler {
         self.bench.borrow().resolve_address(addr)
     }
 
-    ///// Private helpers /////
+    /// Returns a blockstore wrapper that can be passed to functions requiring a generic BS: Blockstore
+    /// It clones the underlying Rc<Bench> so that it can produce a valid &dyn Blockstore reference
+    /// even after this function returns
```

Comment:
Ok, now that I see the ownership implications more clearly, I take it back. Let's expose the `bench` to be borrowed, but via a method `bench()` if possible. This means the BlockstoreWrapper goes on the outside, but we can still make a concise function to wrap the &Bench.